packages/mantine/src/styles/Tabs.module.css

```diff
@@ -1,45 +1,13 @@
-.list {
-    &[data-orientation='horizontal'] {
-        &::before {
-            border-style: none;
+.root {
```

Comment:
Not sure exactly what change caused this, but I noticed a difference here

Before
<img width="928" alt="image" src="https://github.com/user-attachments/assets/854a46de-de37-4d27-8371-95b96d93102d">


After
<img width="975" alt="image" src="https://github.com/user-attachments/assets/8be22a88-f935-40ec-9775-4c68ee2dc5e0">

(there is a double border)